PayPalNativePayments/src/main/java/com/paypal/android/paypalnativepayments/PayPalNativeCheckoutRequest.kt

```diff
@@ -13,5 +13,14 @@ data class PayPalNativeCheckoutRequest(
      * Optional: User email to initiate a quicker authentication flow
      * in cases where the user has a PayPal Account with the same email.
      */
-    val userAuthenticationEmail: String? = null
+    val userAuthenticationEmail: String? = null,
+
+    /**
+     * @property [hasUserLocationConsent] informs the SDK if your application has obtained
+     * consent from the user to collect location data in compliance with
+     * <a href="https://support.google.com/googleplay/android-developer/answer/10144311#personal-sensitive">
+     * Google Play Developer Program policies</a>
+     * This flag enables PayPal to collect necessary information required for Fraud Detection and Risk Management.
+     */
+    val hasUserLocationConsent: Boolean = false
```

Comment:
I think we'll want to eventually make this a required field. I'm not sure if there's a way to deprecate the constructor that does not pass in this boolean 🤔

Reply:
As we'd want a non-breaking change, we provide a default value.
For the next version, we'd want to remove the default value so the merchant developer has to pass in the value for `hasUserLocationConsent`.

Because the request object is the same, we don't have to change the constructor itself.

Reply:
Yep! we'd definitely want to keep the default value for this minor version update. But I'm wondering if there's a way we can deprecate the constructor that does not include the `hasUserLocationConsent` just so merchants have a heads up before the property is removed.

Reply:
>But I'm wondering if there's a way we can deprecate the constructor that does not include the hasUserLocationConsent just so merchants have a heads up before the property is removed.

I'm not sure I understand this statement.

>the constructor that does not include the hasUserLocationConsent

can you please point me to what you are referring to ^?

The constructor `startCheckout(request: PayPalNativeCheckoutRequest)` accepts a request object. I'm adding the boolean to the request object instead of the constructor itself.

Reply:
Oh i was referring to the `PayPalNativeCheckoutRequest` constructor, not the `startCheckout()` function.

I was wondering if there was a way to flag when a merchant creates a `PayPalNativeCheckoutRequest` instance without passing `hasUserLocationConsent`.

Reply:
I had a brain fart. :/